go.mod

```
@@ -3,70 +3,75 @@ module github.com/smartcontractkit/chainlink-terra
 go 1.19
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.5
-	github.com/pelletier/go-toml v1.9.4
+	github.com/CosmWasm/wasmd v0.29.1
+	github.com/cosmos/cosmos-sdk v0.45.9
+	github.com/pelletier/go-toml v1.9.5
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.1
+	github.com/prometheus/client_golang v1.13.0
 	github.com/shopspring/decimal v1.2.0
 	github.com/smartcontractkit/chainlink-relay v0.1.6-0.20220824173548-dab4ac0a6595
 	github.com/smartcontractkit/libocr v0.0.0-20220812191430-db92a9fdaa52
 	github.com/smartcontractkit/terra.go v1.0.3-0.20220108002221-62b39252ee16
-	github.com/stretchr/testify v1.7.1
-	github.com/tendermint/tendermint v0.34.15
-	github.com/terra-money/core v0.5.20
+	github.com/stretchr/testify v1.8.0
+	github.com/tendermint/tendermint v0.34.21
+	github.com/terra-money/core v0.5.2
 	go.uber.org/multierr v1.8.0
 	go.uber.org/ratelimit v0.2.0
 	go.uber.org/zap v1.21.0
-	golang.org/x/crypto v0.0.0-20220210151621-f4118a5b28e2
+	golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90
```

How about newer & semver?
```suggestion
	golang.org/x/crypto v0.1.0
```